projects/chaos/inc/gpio_seq.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+// GPIO event sequencing with delays between actions.
+// Requires initialization of gpio, soft_timer and interrupts.
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "gpio.h"
+#include "status.h"
+
+// Sets the delay time between updating pins, defaults to no delay.
+void gpio_seq_set_delay(uint32_t delay_us);
```

Comment:
I think it would be more obvious if the delay was passed in at each function and I don't think we gain much from making it a static variable. Thoughts?

Reply:
Done. Thanks for the clarification, most of the Relays are actuated over CAN IIRC so they won't use this but this is definitely a better API suggestion.